pkgs/by-name/bo/boxflat/package.nix

```
@@ -0,0 +1,127 @@
+{
+  lib,
+  fetchFromGitHub,
+  gtk4,
+  libadwaita,
+  python3Packages,
+  gobject-introspection,
+  wrapGAppsHook4,
+  copyDesktopItems,
+  makeDesktopItem,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "boxflat";
+  version = "1.27.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Lawstorant";
+    repo = "boxflat";
+    tag = "v${version}";
+    hash = "sha256-W+pnoSyTJS+ORb8vHX4QXOxkDskTb7X2cv6WyU3ctGQ=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  propagatedBuildInputs = [
+    gtk4
+    libadwaita
+
+    python3Packages.pyyaml
+    python3Packages.psutil
+    python3Packages.pyserial
+    python3Packages.pycairo
+    python3Packages.pygobject3
+    python3Packages.evdev
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    wrapGAppsHook4
+    gobject-introspection
+  ];
+
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace-fail "psutil==6.1.0" "psutil==6.1.1" \
+      --replace-fail "evdev==1.7.1" "evdev==1.9.0"
```

This is useless, we want to relax them way more.

```suggestion
      --replace-fail "psutil==6.1.0" "psutil=" \
      --replace-fail "evdev==1.7.1" "evdev"
```